src/components/SuggestionsList/SuggestionsList.test.tsx

```diff
@@ -59,51 +60,50 @@ describe('<SuggestionsList />', () => {
               title: 'groupTitle',
               items: [
                 {
+                  template: SuggestionsListItemTemplate.TEXT,
                   id: 'id',
-                  imageUrl: 'imageUrl',
                   name: 'name',
-                  description: 'description',
                 },
               ],
             },
           ],
         };
-        rendered = shallow(<SuggestionsList {...getProps(options)} />);
+        component = shallow(<SuggestionsList {...getProps(options)} />);
       });
 
-      it('renders as expected', () => {
-        expect(rendered).toMatchSnapshot();
+      it('matches its snapshot', () => {
+        expect(component).toMatchSnapshot();
       });
 
       describe('when onHover is called', () => {
         beforeEach(() => {
-          rendered.find('SuggestionsListItem').simulate('hover', '1');
+          component.find('SuggestionsListItem').simulate('hover', '1');
         });
 
         it('updates the state', () => {
-          expect(rendered.state()).toMatchSnapshot();
+          expect(component.state()).toMatchSnapshot();
```

Comment:
I know you didn't do this but on reflection snapshotting state is mostly a smell to me - we should care about side effects (in this case the rendered output). The internal state representation is an implementation detail. I think it's fine to keep it but just thinking out loud

Reply:
ah i saw this too and meant to circle back to it. i don't know why this test is here because this component doesn't even track any state. i will remove it